browser/ui/brave_browser_command_controller.cc

```diff
@@ -153,7 +154,9 @@ void BraveBrowserCommandController::UpdateCommandForBraveSync() {
 }
 
 void BraveBrowserCommandController::UpdateCommandForBraveWallet() {
-  UpdateCommandEnabled(IDC_SHOW_BRAVE_WALLET, true);
+  UpdateCommandEnabled(
+      IDC_SHOW_BRAVE_WALLET,
+      browser_->profile()->GetPrefs()->GetBoolean(kBraveWalletEnabled));
```

Comment:
Is this correct? We only want to show `Crypto Wallets` if you have it enabled? (for active profile)

Reply:
nervmind - I see the check below in `brave_app_menu_model.cc`; this is a lot cleaner 😄